bin/tx-prover/Cargo.toml

```diff
@@ -46,13 +42,18 @@ pingora-limits = "0.3"
 [dependencies]
 async-trait = "0.1"
 axum = {version = "0.7", optional = true }
+clap = { version = "4.5", features = ["derive"] }
+figment = { version = "0.10", features = ["toml", "env"] }
 miden-lib = { workspace = true, default-features = false }
 miden-objects = { workspace = true, default-features = false }
 miden-tx = { workspace = true, default-features = false }
 prost = { version = "0.13", default-features = false, features = ["derive"] }
 rand = "0.8"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0", features = ["raw_value"] }
```

Comment:
Do we use JSON serialization somewhere?

Reply:
No, we don't. I'm deleting this dependency.